spec/features/admin/campagne_spec.rb

```diff
@@ -160,4 +160,25 @@
       end
     end
   end
+
+  describe 'suppression' do
+    context 'en superadmin' do
+      before do
+        compte_conseiller.update(role: 'superadmin')
+        evaluation.destroy
+      end
+
+      context 'quand je supprime une campagne avec des évaluations associées soft_deleted' do
```

Comment:
Pas besoin de précisé "soft_deleted" ici

```suggestion
      context 'quand je supprime une campagne avec des évaluations associées' do
```